plugins/parsers/json_v2/parser.go

```diff
@@ -53,21 +62,26 @@ type JSONObject struct {
 	IncludedKeys       []string          `toml:"included_keys"`        // OPTIONAL
 	ExcludedKeys       []string          `toml:"excluded_keys"`        // OPTIONAL
 	DisablePrependKeys bool              `toml:"disable_prepend_keys"` // OPTIONAL
+	FieldPaths         []DataSet         // OPTIONAL
+	TagPaths           []DataSet         // OPTIONAL
 }
 
 type MetricNode struct {
-	OutputName  string
-	SetName     string
-	Tag         bool
-	DesiredType string // Can be "int", "uint", "float", "bool", "string"
+	ParentIndex       int
+	OutputName        string
+	SetName           string
+	Tag               bool
+	DesiredType       string      // Can be "int", "uint", "float", "bool", "string"
+	IncludeCollection *PathResult // If set to true, it should be auto included
```

Comment:
The comment "If set to true" is referring to the bool PathResult.tag or just if IncludeCollection is not nil?

Reply:
Good catch, I am not sure what I meant by this. It must have been a boolean originally, I've removed it and added a more descriptive comment that hopefully is more clear.